packages/near-api-js/src/account.ts

```diff
@@ -710,4 +711,100 @@ export class Account {
             total: summary.total.toString(),
         };
     }
+
+    createTransaction(recevier: Account | string): Transaction {
+        return new Transaction(this, recevier);
+    }
+  
+    async hasDeployedContract(): Promise<boolean> {
+        return (await this.state()).code_hash !== EMPTY_CONTRACT_HASH;
+    }
 }
+
+
+/**
+ * Transaction Builder class. Initialized to an account that will sign the final transaction
+ */
+export class Transaction {
+    readonly receiverId: string;
+    readonly actions: Action[] = [];
```

Comment:
Is there a use case for clearing the set of actions? Could it be done automatically somewhere?

Reply:
Moving this to inside the constructor to be more explicit 👍